revenuecat_examples/purchase_tester/android/app/build.gradle

```diff
@@ -38,6 +38,13 @@ android {
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        // purchases-hybrid-common has two variants, but this library doesn't have any
+        // Gradle doesn't know which variant of purchases-hybrid-common to use when using
+        // includeBuild and working with a local copy of purchases-hybrid-common
+        // This line also needs to be added to purchases_flutter's build.gradle
+        // https://developer.android.com/studio/build/build-variants#resolve_matching_errors
+        missingDimensionStrategy 'dependencyVersions', 'latestDependencies', 'unityIAP'
```

Comment:
just to be clear: for customers, this won't be an issue, because `purchases_flutter`s gradle package provides the `missingDimensionStrategy`, right? 
Why is this needed in the sample app?

Reply:
This is only an issue when pointing to a local copy of purchases-hybrid-common, since that project has two variants. When we publish `purchases-hybrid-common` we publish the variants separately as different artifacts so this is not an issue for customers

Reply:
that makes sense, thanks!